src/lib/Controller.js

```diff
@@ -90,18 +90,6 @@ export default class Controller {
       })
     })
 
-    // migrate from extension storage to indexedDB
-
-    browser.storage.local
-      .get('accounts')
-      .then(async d => {
```

Comment:
We'll need another migration like this from localforage to normal storage, so no accounts get lost.

Reply:
done in 0e9fb1e21b3e6d4e6f3442930f5e2cb84212a7d2